index.html

```diff
@@ -4292,6 +4306,493 @@ <h3>Proof Formats</h3>
+If present, ensure that the `refreshService` property conforms to
+the "MUST" statements in Section <a href="#refreshing"></a>.
+                  </li>
+                  <li>
+If present, ensure that the `evidence` property conforms to
+the "MUST" statements in Section <a href="#evidence"></a>.
+                  </li>
+                </ol>
+              </li>
+            </ol>
+          </li>
+          <li>
+If no errors were detected to this point, set
+<var>dataModelVerified</var>.<var>document</var> to <var>inputDocument</var>,
+<var>dataModelVerified</var>.<var>verified</var> to `true`.
+          </li>
+          <li>
+Return <var>dataModelVerified</var>.
+          </li>
+        </ol>
+
+      </section>
+
+      <section class="normative">
+        <h3>Verify Securing Mechanism</h3>
+
+        <p>
+The algorithm in this section can be used to verify whether a securing mechanism
+that is associated with a sequence of bytes is valid. This algorithm takes a
+byte sequence (bytes <var>inputBytes</var>) and a media type
+(string <var>inputMediaType</var>) as inputs, and produces an object
+(<var>securingMechanismVerified</var>) that contains the following:
+        </p>
+
+        <ul>
+          <li>
+a verification status (boolean <var>status</var>)
+          </li>
+          <li>
+a verified document (object <var>document</var>)
+          </li>
+          <li>
+a media type (string <var>mediaType</var>)
+          </li>
+          <li>
+zero or more warnings (array of <a>ProblemDetails</a> <var>warnings</var>)
+          </li>
+          <li>
+zero or more errors (array of <a>ProblemDetails</a> <var>errors</var>)
+          </li>
+        </ul>
+
+        <p>
+The securing mechanism verification algorithm is as follows:
+        </p>
+
+        <ol class="algorithm">
+          <li>
+Set <var>securingMechanismVerified</var> to an empty object and initialize the
+following fields: <var>status</var> to `false`, <var>warnings</var> to an
+empty array, and <var>errors</var> to an empty array.
+          </li>
+          <li>
+Determine the specific mechanism for extracting the <a>conforming document</a>
+(<var>extractDocument</var>):
+            <ol class="algorithm">
+              <li>
+If the <var>inputMediaType</var> is `application/vc+ld+json` or
+`application/vp+ld+json`, the <var>extractDocument</var> function takes
+<var>inputBytes</var> as an input parameter, parses the value as JSON into
+an object, removes the `proof` property, and returns the result as an object.
+If JSON parsing fails, a <a href="#PARSING_ERROR">PARSING_ERROR</a> is appended
+to <var>securingMechanismVerified</var>.<var>errors</var>.
+              </li>
+              <li>
+If the <var>inputMediaType</var> is associated with the JOSE or COSE
+specifications, the [[?VC-JOSE-COSE]] specification defines a process for
+extracting the <a>conforming document</a>.
+<div class="issue" title="Document extraction process in vc-jose-cose">
```

Comment:
We don't intend to document this process, because it does not make any sense.

We do have issues to track defining verification more concretely:

- https://github.com/w3c/vc-jose-cose/issues/187

Reply:
I suggest this issue marker be reworked.

Reply:
@msporny waiting for you to address this, file an issue, or state clearly that you don't plan to make changes.

Reply:
> [@OR13] I suggest this issue marker be reworked.

Please suggest a "reworking", which would obviously conform with what you want it to say, to which we could then respond directly, rather than any of us guessing at what you want it to say, awaiting another "rework it" from you, and guessing again.